Broadcom/symantec-endpoint-protection/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,52 @@
+[{
+    "value": "Allowed execution of {file.path} on {host.hostname} ({host.ip})",
+    "conditions": [
+        {
+            "field": "event.reason",
+            "value": "SONAR detection now allowed"
+        },
+        {
+            "field": "host.hostname"
+        },
+        {
+            "field": "host.ip"
+        },
+        {
+            "field": "file.path"
+        }
+    ],
+    "relationships": [
+        {
+            "source": "host.hostname",
+            "target": "file.path",
+            "type": "execute"
+        }
+    ]
+},{
+    "value": "Virus found on {host.hostname} ({host.ip}): {file.path} was {event.action}",
+    "conditions": [
+        {
+            "field": "event.reason",
+            "value": "Virus found"
+        },
+        {
+            "field": "host.hostname"
+        },
+        {
+            "field": "host.ip"
+        },
+        {
+            "field": "file.path"
+        },
+        {
+            "field": "event.action"
+        }
+    ],
+    "relationships": [
+        {
+            "source": "host.hostname",
+            "target": "file.path",
+            "type": "execute"
```

Comment:
```suggestion
            "type": "executed"
```

Reply:
was the file really executed or it is like a file-system scan that detected a malicious file ?